src/graph/FetchVerticesExecutor.cpp

```diff
@@ -133,27 +133,27 @@ void FetchVerticesExecutor::processResult(RpcResponse &&result) {
     std::unique_ptr<RowSetWriter> rsWriter;
     auto uniqResult = std::make_unique<std::unordered_set<std::string>>();
     for (auto &resp : all) {
-        if (!resp.__isset.vertices || !resp.__isset.vertex_schema
-                || resp.get_vertices() == nullptr || resp.get_vertex_schema() == nullptr) {
+        if (!resp.__isset.vertices) {
```

Comment:
why remove another check ?

Reply:
This is because we have moved the schema to tag_data.